src/utils.ts

```diff
@@ -123,6 +128,118 @@ export function extensionConfigurationSet<
   return getConfiguration().update(key, value, target);
 }
 
+export function canDownloadLanguageServer(client: LanguageServerClient): boolean {
```

Comment:
Can't you use LanguageServerClient::donwloadInfo() instead of having this function? Perhaps it just got left by accident